recipes/r-pdftools/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = '1.8' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pdftools
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: pdftools_{{ version }}.tar.gz
+  url:
+    - {{ cran_mirror }}/src/contrib/pdftools_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pdftools/pdftools_{{ version }}.tar.gz
+  sha256: 5f790c3678eacdc15147a671571c86b5f8d6eb692fa680324bc5d27542e6d0fb
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('cxx') }}      # [not win]
+    - {{native}}toolchain        # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}sed               # [win]
+    - {{posix}}grep              # [win]
+    - {{posix}}autoconf
+    - {{posix}}automake-wrapper  # [win]
+    - {{posix}}automake          # [win]
+    - {{posix}}pkg-config
+    - {{posix}}make
+    - {{posix}}sed  # [win]
+    - {{posix}}coreutils  # [win]
+
+  host:
+    - r-base
+    - r-rcpp >=0.12.12
```

Comment:
If [poppler](https://github.com/conda-forge/poppler-feedstock) is the dependency that circle-ci is complaining about ("Configuration failed because poppler-cpp was not found."), adding it here might get the build to progress.

Reply:
@loriab did I put that in there wrong? Or is there some other issue? Not sure why the one addition would have made the linter so unhappy.